app/components/NavigationTab/NavigationLink.tsx

```diff
@@ -1,32 +1,44 @@
 import cx from 'classnames';
-import { NavLink } from 'react-router-dom-v5-compat';
+import { NavLink, useLocation } from 'react-router-dom';
 import styles from './NavigationLink.css';
-import type { ReactNode } from 'react';
+import type { MouseEventHandler, ReactNode } from 'react';
 
 type Props = {
-  to: string;
-  onClick?: (e: Event) => void;
+  to: string | string[];
```

Comment:
Initially didn't find it that intuitive that the remainder of the `to`-array is not used for navigating and only to decide `isActive`. Could it be considered to change the params to something like the following?
```suggestion
  to: string;
  activePaths?: string[];
```

Otherwise clean solution for the link 💯

Reply:
Yes that's probably more intuitive

Reply:
I'm thinking of calling it `additionalActivePaths` though and let it accept "additional" paths so that you don't have to write up the "main" path twice - less error prone I suppose.

Reply:
Absolutely, I was thinking the exact same thing - but your wording is better